app/js/filters.js

```diff
@@ -16,7 +17,7 @@ angular.module('myApp.filters', ['myApp.i18n'])
       if (!user || !user.first_name && !user.last_name) {
         return _('user_name_deleted')
       }
-      return user.first_name + (user.last_name ? ' ' + user.last_name : '')
+      return user.first_name ? (user.first_name + (user.last_name ? ' ' + user.last_name : '')) : user.last_name
```

Comment:
afaik it's not possible to have no first_name but have a last_name, so this isn't really needed.